apps/mobile/src/assets/swap.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="25" height="24" viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Do you think we can share these `svg`'s better with web? 

We have icon components for this in the monorepo now - `swap-icon.tsx`. Can we use that in ReactNative?

Or if not maybe we need to refactor our icon components to abstract the `.svg` code so we can share?